### DISCUSSION TITLE
What is the best practices for routing permissions?

### DISCUSSION THREAD

**Discussion by s3xysteak** [posva/unplugin-vue-router#454] [Questions and Help] [answered]
Before the plugin, I use dynamic routes to achieve the routing permission. The logic is in the backend, a user can only access routes for which he has permissions, which based on `addRoute`. How could I achieve the similar target with `unplugin-vue-router`?

**Comment by posva** [comment, marked as answer]
The best practice for route permissions is to actually just use a navigation guard, without dynamic routing as it doesn't add any security by itself.
